packages/app/src/systems/Core/components/Layout.tsx

```diff
@@ -1,18 +1,45 @@
+import type { ThemeUtilsCSS } from '@fuel-ui/css';
 import { cssObj } from '@fuel-ui/css';
 import { Box } from '@fuel-ui/react';
-import type { ReactNode } from 'react';
+import type { FC, ReactNode } from 'react';
 import { Helmet } from 'react-helmet';
 
+import { coreStyles } from '../styles';
+
 import { Header } from './Header';
 
 import { META_DESC, META_OGIMG } from '~/constants';
+import { OverlayDialog } from '~/systems/Overlay';
+
+type ContentProps = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  as?: any;
+  children: ReactNode;
+  css?: ThemeUtilsCSS;
+};
+
+const Content = ({ as, children, css }: ContentProps) => {
+  return (
+    <Box
+      as={as}
+      css={{ ...styles.content, ...css }}
+      className="layout__content"
+    >
+      {children}
+    </Box>
+  );
+};
+
+type LayoutComponent = FC<LayoutProps> & {
+  Content: typeof Content;
+};
 
 type LayoutProps = {
   title?: string;
   children: ReactNode;
 };
 
-export function Layout({ title, children }: LayoutProps) {
+export const Layout: LayoutComponent = ({ title, children }: LayoutProps) => {
```

Comment:
Do we need `: LayoutComponent` here?

Reply:
to export the `Layout.Content` together with good typing